node/resolvers/catalog/product.ts

```diff
@@ -57,6 +75,22 @@ export const resolvers = {
 
     recommendations: product => product,
 
-    titleTag: ({productTitle}) => productTitle,
+    titleTag: ({ productTitle }) => productTitle,
+
+    specificationGroups: product => {
+      const allSpecificationsGroups = product.allSpecificationsGroups.concat([
```

Comment:
I tried the query and an error was thrown:

> "Cannot read property 'concat' of undefined",

https://highlightsprod--storecomponents.myvtex.com/_v/vtex.store-graphql@2.60.7/graphiql/v1?operationName=Product&variables=%7B%0A%20%20%22slug%22%3A%20%22long-sleeve-shirt%22%0A%7D&query=query%20Product(%24slug%3A%20String)%20%7B%0A%20%20product(slug%3A%20%24slug)%20%7B%0A%20%20%20%20specificationGroups%20%7B%0A%20%20%20%20%20%20name%0A%20%20%20%20%20%20specifications%20%7B%0A%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D

The code that I pasted was just and example hehe so you might not to take into account empty fields